project/src/main/java/com/google/sps/data/Event.java

```diff
@@ -186,25 +193,41 @@ public String getLocation() {
     return this.location;
   }
 
-  public LocalDate getDate() {
+  public Date getDate() {
     return this.date;
   }
 
   public Set<VolunteeringOpportunity> getOpportunities() {
     return this.opportunities;
   }
 
+  public Set<Long> getOpportunitiesIds() {
```

Comment:
Liangqi suggested on my PR to do something similar to this for this func:
`return opportunities.stream().map(VolunteeringOpportunity::getId).collect(Collectors.toSet());`

Reply:
beautiful, thank you, done